I.write-tests/05-get-second-third-smallest/get-second-third.js

```diff
@@ -1,7 +1,7 @@
-module.exports = function (array) {
-    const newArray = array.slice()
-    newArray.sort(function (x, y) {
-        return x > y;
-    });
-    return [newArray[1], newArray[2]];
-};
\ No newline at end of file
+module.exports = function(array) {
+  const newArray = array.slice();
+  newArray.sort(function(x, y) {
+    return x > y;
```

Comment:
So I tested this implentation in Node **11.8.0** and it fails, while in prior versions it passes.
Maybe we can change it to `x - y` which works with all versions?

Reply:
🤦‍♂️ That's because I messed up the implementation. That'll teach me for not running the tests 😄